### PR TITLE
Add .gitattributes information on generated files 🍥

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This should help not displaying the diff of those generated files by
default.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This file is documented at https://git-scm.com/docs/gitattributes.
+# Linguist-specific attributes are documented at
+# https://github.com/github/linguist.
+
+**/zz_generated.*.go linguist-generated=true
```
Comment (afrittoli) [inline .gitattributes]: According to the README at https://github.com/github/linguist, the`=true` part is not needed.

Comment (vdemeester) [reply .gitattributes]: oh :sweat_smile: I used the same as on `tektoncd/pipeline` :stuck_out_tongue:

**Pull request merged**